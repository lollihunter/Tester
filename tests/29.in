3 20 20
1 1
1 16
1 5
1 19
3 18
1 12
2 10
3 11
1 9
2 19
2 9
2 5
1 4
2 2
1 6
2 12
3 1
1 15
2 20
3 20
