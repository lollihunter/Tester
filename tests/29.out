12
1 14 13 3 15 11 7 6 18 2 5 20 
